package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "login-forms",
+  "version": "1.0.0",
```

Comment:
I feel like this should be a sub-1.0.0 note for now. Perhaps `0.0.1`?

Reply:
Wait... this isn't composer - how does package.json versioning work for something that is never 'published'? Is it even necessary?

Reply:
https://docs.npmjs.com/files/package.json#version:
> If you don't plan to publish your package, the name and version fields are optional.